docker/generate_makefile.py

```diff
@@ -56,16 +56,26 @@ def _print_makefile_run_template(image):
 \t-e FUZZ_TARGET=$({benchmark}-fuzz-target) \\\
 '.format(fuzzer=fuzzer, benchmark=benchmark))
 
+        params = ""
         if run_type == 'test-run':
             print('\t-e MAX_TOTAL_TIME=20 \\\n\t-e SNAPSHOT_PERIOD=10 \\')
         if run_type == 'debug':
             print('\t--entrypoint "/bin/bash" \\\n\t-it ', end='')
+        elif run_type == 'repro-check':
+            if os.path.isdir(os.path.join(BENCHMARK_DIR, benchmark,
+                                          'testcases')):
+                print('\t--entrypoint /bin/bash ', end='')
+                params = " -c \"for f in \\$$OUT/testcases/*; do\
+                        \\$$OUT/\\$$FUZZ_TARGET -timeout=25\
+                        -rss_limit_mb=2560 \\$$f; done;\""
```

Comment:
Where does `2560` come from? Is it because some of the testcases from Matio cause an OOM with 2048?

Reply:
No, it's from OSS-Fuzz, docs say that the memory limit while fuzzing is 2.5GB